code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -644,6 +648,21 @@
 					M.adjustBruteLoss(rand(0,15))
 					M.adjustToxLoss(rand(0,15))
 					M.adjustFireLoss(rand(0,15))
+					if(ishuman(M) && M.mind && prob(10)) // To prevent people from using this on monkies to cause pandemics
+						var/mob/living/carbon/human/H = M
+						// Their body is being suffused with magical life-inducing stuff - so other things get lively, too
+						// Things are going to get interesting for the reviving party, too
+						var/virus_type = pick(list(
+							/datum/disease/advance/flu,
+							/datum/disease/advance/cold,
+							/datum/disease/brainrot,
+							/datum/disease/magnitis,
+							/datum/disease/fake_gbs, // honk
+							/datum/disease/cold9
+						))
+						var/datum/disease/D = new virus_type()
+						H.AddDisease(D)
```

Comment:
I think you're missing the point. Individual choices, on the whole should largely have individual costs, both for the person involved and the person on the receiving end of it. To that degree, all methods of revival fit within this scope. This is very different from that; it's introducing a medical-bay wide penalty) or even station wide if the person makes a mistake for the action of bringing a singular person back into the game.

Defibs work for everyone within a 5 minute window, without any negative consequences.

SR works for an unlimited duration and deals some post revive damage (albeit not much). 

Does it need tweaking? Probably so, in comparison to a defib, but externalizing the cost of going with this over another method of revival isn't the way to go about adding a downside to player agency. Individual pre and post consequences (damage from being revived, volume requirement, difficulty in acquisition, time limits, or useage requirements) are better tuning levels than group levers.